contrib/selinux/src/syslog_ng.el67.te.in

```diff
@@ -1 +0,0 @@
-
```

Comment:
This file is referenced in `contrib/Makefile.am` and that's why [github actions](https://github.com/syslog-ng/syslog-ng/runs/2202651887?check_suite_focus=true) tarball job fails.
Please check the filenames in `contrib/Makefile.am` and rename or delete them accordingly.

I think this is the reason why `make distcheck` fails too in [github-actions](https://github.com/syslog-ng/syslog-ng/pull/3627/checks?check_run_id=2202651727)

Reply:
Thanks Gábor! I updated the Makefile.am file to reflect the changes in the renamed/moved files.